### ISSUE TITLE
The -y flag for `jupyter notebook` isn't working for me.

### ISSUE THREAD

**Issue by Fitzy1293** [jupyter/notebook#6223]
**Describe the bug**

The `-y` flag for `jupyter notebook` does not seem to work. I was prompted for for an answer when doing a keyboard interrupt.

**To Reproduce**

- run `jupyter notebook -y` (or `jupyter-notebook -y`)
- Hit ctrl+c
- Still get a prompt for yes or no.

**Expected behavior**

The `-y` flag is supposed to automatically make all questions default to yes.
This is to make things scriptable, and easier to use if you just want default behavior, I assume.   

**Screenshots**

![help](https://imgur.com/gmy3Kx1.png)

![example](https://i.imgur.com/yDybfXK.png)

**Desktop (please complete the following information):**

 - OS: Ubuntu 20.04
 - Browser chromium (snap install)
 - Version 95.0.4638.69 chromium
 - `jupyter notebook` --version = 6.0.3

**Additional context**

I've tried to come up with a way to get stuff to work better for me.
Now I have the script below mapped to a shortcut, it opens jupyter in the browser without any blocking behavior.
It launches in whatever is the current directory.

I tried to make it only have the info I needed print out.

Here's what it looks like when you run it.

![script](https://i.imgur.com/e56D6vz.png)

```shell
#!/bin/sh

# Script to get `jupyter notebook` to work better on startup.
# Wanted to add it to a keyboard shortcut, but it doesn't auto open in the browser, so this is a workaround.

if [ $# -eq 0 ]; then
	echo  "usage:
  launchjupyter start                    starts a jupyter browser session
    --test                               run script without opening the browser"
	exit 1
fi


kill_running_notebooks() {
	# Cleanup older files in the runtime folder.
	run_dir="$HOME/.local/share/jupyter/runtime"

    # Kill old notebooks on startup.
	# Hacky way of killing the processes, but wanted to see what I was actually ending.
	killcmd="\"$(pgrep -d "\" \"" -f jupyter-notebook)\""

	if  echo "$killcmd" | grep -q "[0-9]"; then
		rm -rf "$run_dir" && mkdir "$run_dir"

		echo "previous notebooks found running"
		pgrep -fa "jupyter-notebook" | cat -n
		printf "\trunning: eval kill $killcmd\n"
		eval "kill $killcmd"

	else echo "no old notebooks found, starting new kernel"
	fi
    return 0
}


runjupyter() {
    # Run a notebook detatched from the terminal.
	nohup jupyter-notebook > /dev/null 2>&1 &
	return 0
}


run_script () {
	# Kill the old ones, start a new one.
	kill_running_notebooks && runjupyter

	# Get new notebook url
	url="$(jupyter notebook list | grep "http" | cut -d ' ' -f 1)"
	printf "local-url:%s\n" "$url"

	# Check if we used the test flag or not, if not open in the browser.
    echo "$@" | grep -q -v "\-\-test" && nohup python3 -m webbrowser -t "$url" > /dev/null 2>&1 &

	echo "target:$PWD"
	printf "new-process:%s\n" "$(pgrep -af "jupyter-notebook")"

	return 0
}

# Only run if $1 is start.
if [ "$1" = "start" ]; then run_script "$@"; fi
 exit 0
```

**Comment by Fitzy1293** [comment]
`printf "local-url%s\n" "$url"`
Should be - 
`printf "local-url:%s\n" "$url"`
Forgot the the semicolon after "local-url". I'll fix that now.

**Comment by kevin-bates** [comment]
Please upgrade to a more recent release of notebook.  This issue was resolved by https://github.com/jupyter/notebook/pull/5941 and is available in notebook releases >= 6.3.0. Your output shows notebook 6.0.3.

**Comment by Fitzy1293** [comment]
> Please upgrade to a more recent release of notebook. This issue was resolved by #5941 and is available in notebook releases >= 6.3.0. Your output shows notebook 6.0.3.

Okay I will do that,  thank you.
